src/newsletter-editor/editor/style.scss

```diff
@@ -2,7 +2,9 @@
 	.editor-post-title {
 		&.newspack-newsletters-post-title-hidden {
 			height: 0;
+			margin: -4rem 0 calc( -1 * var( --wp--style--block-gap ) );
```

Comment:
This negative margin causes the content to get cut-off at the top if your layout doesn't have the logo image as the first block:

<img width="1786" alt="Screen Shot 2022-02-01 at 4 41 10 PM" src="https://user-images.githubusercontent.com/2230142/152070011-bd9c583b-596c-4ee6-8d1b-676f1d89bf85.png">

Reply:
Fixed: https://github.com/Automattic/newspack-newsletters/pull/710/commits/47f2f4e74cab7ae11fa951c1ab6b719eccb2ac8b